hail/Makefile

```diff
@@ -234,11 +241,18 @@ pypi-deploy: check-pypi test-dataproc set-docs-sha
 TAG_EXISTS = $(shell git ls-remote --exit-code --tags origin $(HAIL_PIP_VERSION) || echo "does not exist")
 .PHONY: check-tag
 check-tag:
-	if [ -z "$(TAG_EXISTS)" ]; then echo "tag $(HAIL_PIP_VERSION) already exists"; exit 1; fi
+	if [ -z "$(TAG_EXISTS)" ]; then echo "ERROR: tag $(HAIL_PIP_VERSION) already exists"; exit 1; fi
+
+annotation_db_json_url := gs://hail-common/annotationdb/$(HAIL_PIP_VERSION)/annotation_db.json
+.PHONY: deploy-annotation-db
+deploy-annotation-db:
+	(! gsutil ls $(annotation_db_json_url) >/dev/null 2>/dev/null && \
+    gsutil cp python/hail/experimental/annotation_db.json $(annotation_db_json_url)) || \
```

Comment:
alternatively, can set a temporary hold on those files, like we do here:
https://github.com/hail-is/hail/blob/b55108125f0f857418f3b46f3fe0734ad736c042/build.yaml#L915

Reply:
I intended this expression to prevent overwriting a previously deployed JSON file. For example, if you accidentally run this target on some branch, it should not modify the currently deployed version.

Reply:
gsutil retention prevents anyone from deleting/overwriting the file until someone manually lifts the hold, which does basically the same check you're doing here (I like that check) but more strongly.

Reply:
Oh nice, man the gsutil docs for that really don't make it clear what its doing.